src/templates/index.hbs

```diff
@@ -7,7 +7,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
   <body>
-    {{> favorite presentation='My First Presentation' }}
+    <div>
+      {{test}}
```

Comment:
Can you revert this change? It looks like you were testing things out here

Reply:
yes my bad!